src/lib/constants.js

```diff
@@ -157,5 +157,6 @@ export const METADATA = {
 export const ERROR_CODE_403_FORBIDDEN_BY_POLICY = 'forbidden_by_policy';
 
 // LocalStorage Keys
+export const ANNOTATION_COLOR_KEY = 'annotation-color';
```

Comment:
If this is only used in one file, can we encapsulate it there?

Reply:
Yes that works.